rest-api/etl/setup_local_database_for_etl.sh

```diff
@@ -6,7 +6,7 @@
 # the database. (Note: the loading takes upwards of 10 minutes... grab some coffee.)
 #
 # Vocabulary CSV files are copied from GCS at path
-# gs://all-of-us-rdr-vocabulary/vocabularies-2017-09-18/; these files were originally received
+# gs://all-of-us-rdr-vocabulary/vocabularies-2017-11-27/; these files were originally received
```

Comment:
Checking - you intended to update this file without updating `setup_database_for_etl.sh` (which still refers to 9-18)?

Reply:
Heh. Funny you should mention it... I just did that. :)

We may want to change both of these to read from a file which has this in a single place...